src/node.cc

```diff
@@ -2741,6 +2798,16 @@ void SetupProcessObject(Environment* env,
   process->Set(FIXED_ONE_BYTE_STRING(env->isolate(), "execArgv"),
                exec_arguments);
 
+  // process.allowedEnvironmentNodeFlags
+  Local<Array> env_flags = Array::New(env->isolate(),
+                                      arraysize(environment_flags));
+  for (unsigned int i = 0; i < arraysize(environment_flags); ++i) {
+    env_flags->Set(i, String::NewFromUtf8(env->isolate(),
```

Comment:
Can you use `OneByteString()` here or the `NewFromUtf8()` overload that returns a `MaybeLocal<String>`?

Reply:
done